x-pack/test_serverless/functional/test_suites/observability/dataset_quality/dataset_quality_details.ts

```diff
@@ -38,21 +39,24 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
   const excludeKeysFromServerless = ['size']; // https://github.com/elastic/kibana/issues/178954
 
   const apacheAccessDatasetName = 'apache.access';
-  const apacheAccessDatasetHumanName = 'Apache access logs';
+  const apacheAccessDataStreamName = `logs-${apacheAccessDatasetName}-${productionNamespace}`;
```

Comment:
why are we not using the readable name anymore here?

Reply:
Previously readable name was required when clicking on the cell inside the table of the Main page which does not exists any more. Now navigation is happening via URL directly which expects datastream rather than human name